src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift

```diff
@@ -58,6 +58,8 @@ enum AccessibilityIdentifiers {
 			static let antigenCell = "AccessibilityIdentifiers.Home.antigenCell"
 			static let submittedPCRCell = "AccessibilityIdentifiers.Home.submittedPCRCell"
 			static let submittedAntigenCell = "AccessibilityIdentifiers.Home.submittedAntigenCell"
+			static let removeTestButton = "AppStrings.Home.TestResult.ShownPositive.buttonRemoveTest"
+			static let deleteButton = "AppStrings.AntigenProfile.Profile.deleteAlertDeleteButtonTitle"
```

Comment:
```suggestion
			static let deleteAlertDeleteButton = "AppStrings.Home.TestResult.ShownPositive.deleteAlertDeleteButton"
```

I think it needs to be named more precise, otherwise its hard to understand which one is which